challenges/stats_preview_card_component/README.md

```diff
@@ -0,0 +1,125 @@
+# Frontend Mentor - Stats preview card component solution 👋
+
+This is a solution to the [Stats preview card component challenge on Frontend Mentor](https://www.frontendmentor.io/challenges/stats-preview-card-component-8JqbgoU62). Frontend Mentor challenges help you improve your coding skills by building realistic projects.
+
+## Table of contents
+
+- [Frontend Mentor - Stats preview card component solution 👋](#frontend-mentor---stats-preview-card-component-solution-)
+  - [Table of contents](#table-of-contents)
+  - [Overview](#overview)
+    - [The challenge](#the-challenge)
+    - [Screenshot](#screenshot)
+    - [Links](#links)
+  - [My process](#my-process)
+    - [Built with](#built-with)
+    - [What I learned](#what-i-learned)
+    - [**Example:**](#example)
+    - [Continued development](#continued-development)
+    - [Useful resources](#useful-resources)
+  - [Author 🚀](#author-)
+
+## Overview
+
+### The challenge
+
+Users should be able to:
+
+- View the optimal layout depending on their device's screen size
+
+### Screenshot
+
+- Desktop design
+
+![Desktop design preview for the Stats preview card component coding challenge](./design/desktop_design.png)
+
+- Mobile responsive design
+
+![Mobile design preview for the Stats preview card component coding challenge](./design/mobile_design.png)
+
+---
+
+### Links
+
+- Solution URL: [Add solution URL here]()
+- Live Site URL: [Add live site URL here]()
+
+## My process
+
+### Built with
+
+- Semantic HTML5 markup
+- CSS custom properties
+- Flexbox model
+- Mobile friendly responsive design
+- Tailwind utility-first CSS framework (ver. 3+)
+
+### What I learned
+
+- [x] Improved my existing HTML/CSS knownledge
+- [x] Tailwind CSS framework fundamentals, guide principles
+- [x] Valuable picture skills, like image layout with flexbox model, etc.
+
+Configuring and customizing my Tailwind installation:
+
+### **Example:**
+
+> **Creating your own classes:**
+
+```css
+/* your input style.css */
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+@layer components {
+  /* custom css class */
+  .qr-card {
```

Comment:
> Since this `README.md` file is not required I'd recommend you delete it so that you don't have to maintain it. You can leave it if you want, but in that case, remove irrelevant parts.

Sure, I'll remove the from all branches